Added coc.md (#10)
womenwhogo/wikis: coc.md

# Women Who Go - Code of Conduct

> GoBridge (GB) and Women Who Go (WWG) uses the Go CoC as our base document:
> 
> [Go CoC](https://golang.org/conduct) | [Go Forum](https://forum.golangbridge.org/) | [Gophers Slack](http://invite.slack.golangbridge.org) | [Women Who Go Slack](https://womenwhogo.org/invite.html)

Reviewer: Fixes a typo s/uses/use and links to the Go CoC when we mention it in the first sentence:

```suggestion
GoBridge (GB) and Women Who Go (WWG) use the [Go CoC] as our base document:

[Go Forum](https://forum.golangbridge.org/) | [Gophers Slack](http://invite.slack.golangbridge.org) | [Women Who Go Slack](https://womenwhogo.org/invite.html)

[Go CoC]: https://golang.org/conduct
```

It is confusing if our CoC applies to the gobridge forum and slack since they are listed here. I suggest limiting this document to just WWG moderated spaces.

## GB/WWG Addendum

### Real Names
If you feel comfortable, we encourage you to use your real name in all of our online and offline GB/WWG communities and events. We find it encourages openness and civility in discussions. However, this is not a requirement.

Organizers and moderators may ask people to change their handles or usernames if it conflicts with known public figures, impersonates other people, sounds like automated tools, contains inappropriate language, or other unclear cases.

### Language Policy
We are working hard to keep our public spaces family-friendly and as inclusive as possible. At the organizers or moderators discretion, words and images that are deemed inappropriate or offensive will be moderated. Generally, cursing is not allowed. Please reach out to the organizers or moderators for a list of words that are being moderated.

### Moderation

> Participants in our online and offline GB/WWG communities and events should flag inappropriate behavior to organizers and moderators rather than addressing it themselves. This includes but is not limited to reporting: violations of this code of conduct, spam messages, messages posted in the wrong channel or category, and public conversations that are inappropriate. Please report any violations to the #admin-help channel on gophers.slack.com as early as possible or privately message Carmen Andoh or William Kennedy directly. At our offline events please report any violations to the organizers or email support@gobridge.org.

Reviewer: I have just added a channel named #admin-help to our WWG slack so that we can more easily have consistent instructions across our slacks.

I also added my name to the list of people to whom you can report a CoC concern for WWG.

```suggestion
Participants in our online and offline GB/WWG communities and events should flag inappropriate behavior to organizers and moderators rather than addressing it themselves. This includes but is not limited to reporting: violations of this code of conduct, spam messages, messages posted in the wrong channel or category, and public conversations that are inappropriate. Please report any violations to the #admin-help channel slack as early as possible or privately message Carolyn Van Slyck directly. At our offline events please report any violations to the organizers or email hello@womenwhogo.org.
```

#### Expectations for moderator enforcement
While in practice most actions from organizers and moderators will be friendly reminders, they should act on complaints/reports quickly and involve other team members if necessary. They should take appropriate, usually private, action if they see any abuse or harassment. They must be sensitive to cultural differences, time zones, etc., but take prompt action when necessary. An organizer or moderator is one who exists to serve the community.

#### Suggested strategies for moderators when dealing with observed violations
Except in the most egregious cases, organizers and moderators will follow these guidelines:

- First, attempt to privately message or speak with the person and point out to them that their behavior violates the CoC. Include a link or try to be specific about what area of the the CoC has been violated.

> - If the person does not respond to a request or is not willing to talk, escalate to the private #gopher-admins channel on gophers.slack.com or assemble other team members to discuss the incident.

Reviewer: ```suggestion
- If the person does not respond to a request or is not willing to talk, escalate to other leadership members to discuss the incident.
```

We realize spam can be hard to define in some cases, so we will rely on the community or moderators to identify and flag it and handle it appropriately. Obvious spam posts should be promptly deleted; non-obvious spam may be flagged for review. Users are encouraged to flag posts as spam to help bring them to a moderator’s attention.

Not all spam posts are by spam users. For example, a legitimate user account may be compromised. If a moderator notices a reputable user account posting spam, the spam post(s) should be deleted immediately and the user notified. If spam posts continue, the account may be suspended until the user can regain control of his/her account.

Commercial content is allowed if relevant to answering a question or keeping inline with a topic.

#### Reporting guidelines

> If someone is subject to or witnesses unacceptable behavior, or has any other concerns, they are strongly encouraged to notify an organizer, moderator or a GB Leadership team member as soon as possible. Every reported concern will follow this workflow until resolved:

Reviewer: ```suggestion
If someone is subject to or witnesses unacceptable behavior, or has any other concerns, they are strongly encouraged to notify an organizer, moderator or a WWG Leadership team member as soon as possible. Every reported concern will follow this workflow until resolved:
```

> - Make contact by directly talking to an organizer, moderator or emailing support@gobridge.org

Reviewer: ```suggestion
- Make contact by directly talking to an organizer, moderator or emailing hello@womenwhogo.org
```

>   - Your email message will reach the **GB Leadership team** only.

Reviewer: ```suggestion
  - Your email message will reach the **WWG Leadership team** only.
```

>   - Reports will be confidential within the organizers team, moderators team or GB’s Leadership team.

Reviewer: ```suggestion
  - Reports will be confidential within the organizers team, moderators team or WWG’s Leadership team.
```

> - You will receive a response within 24-48 hours (very likely sooner) in our online communities. For our offline communities and events, a response should be expected within an hour or two.

Reviewer: I suggest removing this bullet point as we don't have enough people to give a strong timeline.

Reviewer: I'm not sure if you are keeping this because you wanted to or you missed my comment? I am concerned that we can't meet the SLA listed here.

Author: > I'm not sure if you are keeping this because you wanted to or you missed my comment? I am concerned that we can't meet the SLA listed here.

Oh, I am sorry I missed this comment. I'll get it sorted. Thanks for pointing out :)

- The group will review the incident and determine what happened.
- They may reach out to other community members for more context.
- The team will reach a decision as to how to act. These include:
  - Nothing.

>   - A private reprimand from the GB Leadership team to the individual(s) involved.

Reviewer: ```suggestion
  - A private reprimand from the WWG Leadership team to the individual(s) involved.
```

>   - An imposed vacation (i.e. asking someone to "take a week off" from all GB moderated properties).

Reviewer: ```suggestion
  - An imposed vacation (i.e. asking someone to "take a week off" from all WWG moderated properties).
```

  - A request for a private or public apology.

>   - A temporary or permanent ban from some or all GB moderated communities and events. GB Leadership team will reach out to the original reporter to let them know what action we decided to take.

Reviewer: ```suggestion
  - A temporary or permanent ban from some or all WWG moderated communities and events. WWG Leadership team will reach out to the original reporter to let them know what action we decided to take.
```

>   - Appeals to the decision may be made to the GB Leadership team directly, or any member of the administration team thereof.

Reviewer: ```suggestion
  - Appeals to the decision may be made to the WWG Leadership team directly, or any member of the administration team thereof.
```

#### Administrators
Below is the list of people who are moderators for our online communities. If you have any issues or need help, please feel free to reach out to anyone on this list. There is also the public `admin-help` channel on Slack where people can get help or clarification on any admin related issues.

> **The role of GB (Leadership Team):**

Reviewer: ```suggestion
**The role of WWG Leadership Team:**
```

- They are the stewards for the different online communities.
- They have rights in the systems tied to the primary owner role.
- They act as administrators.

**The role of moderators:**
- They monitor the public forums, channels and threads for inappropriate behavior.
- They have full voting rights for decisions that affect the communities.
- They have rights in the systems tied to the admin role.

**Primary Owner:**  

> GB (support@gobridge.org)

Reviewer: ```suggestion
WWG (hello@womenwhogo.org)
```

> **GB Leadership Team:**  

Reviewer: ```suggestion
**WWG Leadership Team:**  
```

> [Ashley McNamara](https://twitter.com/ashleymcnamara),

Reviewer: Let's list carolyn, maartje and rashmi here.

Reviewer: The WWG leadership team is still showing the GB leadership team.

Author: Sure, Carolyn I'll work on this; couldn't find time to work on it in midst of office-work so thought of completing on Saturday. Hope that's okay? :)

Reviewer: Of course, that's fine! Never any need to explain that this is volunteer work and real work, life, etc always come first.

[Carlisia Pinto](https://twitter.com/carlisia),
[Carmen Hernández Andoh](https://twitter.com/carmatrocity),
[Carolyn Van Slyck](https://twitter.com/carolynvs),
[Conrad Taylor](https://twitter.com/conradwt),
[Diana Ortega](https://twitter.com/dicaormu),
[Edward Muller](https://twitter.com/freeformz), 
[Johnny Boursiquot](https://twitter.com/jboursiquot),
[Maartje Eyskens](https://twitter.com/MaartjeME),
[Natalie Pistunovich](https://twitter.com/NataliePis),
[Sarah Allen](https://twitter.com/ultrasaurus),
[Van Riper](https://twitter.com/vanriper),
[Wilken Rivera](https://twitter.com/wilkenrivera),
[William Kennedy](https://twitter.com/goinggodotnet)

> **Moderators:**  

Reviewer: We don't have moderators.
